SECURITY.md

```diff
@@ -0,0 +1,6 @@
+## Reporting a Vulnerability
+
+If you discover a potential security issue in this project we ask that
+you notify us directly via email to security@oci.run.
```

Comment:
Who is "us"? Who owns `oci.run`?

Reply:
I do, "us" here means "crun maintainers".  `security@oci.run` at the moment is just an alias to my personal email

Reply:
Thanks for the answer, but I feel it is better to document that  OCI does not own `oci.run`.

Reply:
thanks for the review.  Pushed a new version to mention that.